src/Java.Interop.Localization/Resources.resx

```diff
@@ -332,4 +332,51 @@ Name, com.example.MyClass.</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM4002" xml:space="preserve">
+    <value>Please specify at least one ASSEMBLY to process.</value>
+    <comment>The following terms should not be translated or have any capitalization changes: ASSEMBLY. This is a special case for this particular message. In most messages, "assembly" would be translated.</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM4003" xml:space="preserve">
+    <value>Unable to create Java VM{0}{1}</value>
+    <comment>{0} - newline, {1} - exception</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM4004" xml:space="preserve">
+    <value>Unable to read profile file '{0}'.{1}{2}</value>
+    <comment>{0} - path, {1} - newline, {2} - exception. In this message, the term "profile" refers to a customized list of types to process.</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM4005" xml:space="preserve">
+    <value>Path '{0}' does not exist.</value>
+    <comment>{0} - path</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM4006" xml:space="preserve">
+    <value>Unable to process assembly '{0}'{1}{2}{1}{3}</value>
+    <comment>{0} - assembly, {1} - newline, {2} - exception message, {3} exception</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM8001" xml:space="preserve">
+    <value>Couln't find interface {0}</value>
+    <comment>{0} - interface name</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM8003" xml:space="preserve">
+    <value>Catched type Load exception. Make sure all the references are available. For details check the exception: {0}{1}</value>
+    <comment>{0} - newline, {1} - exception</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM8004" xml:space="preserve">
+    <value>Unable to find cecil's TypeDefinition of type {0}. Make sure the paths to all reference assemblies are passed to the tool with the -L option.</value>
+    <comment>{0} - type</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM8005" xml:space="preserve">
+    <value>Marshal methods type '{0}' already exists. Skipped generation of marshal methods in assembly '{1}'. Use -f to force regeneration when desired.</value>
+    <comment>{0} - type, {1} - assembly name. In this message, the term "marshal methods" refers to methods that allow interaction between the managed methods and Java methods, similar to the methods of the .NET System.Runtime.InteropServices.Marshal class.</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM8006" xml:space="preserve">
+    <value>Unable to find cecil's MethodDefinition of method {0}.  Make sure the paths to all reference assemblies are passed to the tool with the -L option.</value>
+    <comment>{0} - method</comment>
+  </data>
+  <data name="JniMarshalMethodGen_JM8007" xml:space="preserve">
+    <value>Unable to find System.Console::WriteLine method. Disabling debug injection. Make sure the paths to corlib assembly is passed to the tool with the -L option.</value>
```

Comment:
I searched for `corlib` on docs.microsoft.com and didn't find any matches, so if it's accurate, maybe `mscorlib` can be used instead. Then if it's also accurate, maybe it's OK to use the C# style of fully qualified method name, and since this message includes a consequence, maybe if it's accurate, it's OK to change the last sentence to reinforce what will happen if the users resolves this warning (that is, it will allow debug injection to be enabled).

All together, and adding a little comment for untranslated terms for safety, maybe:

```
<value>Unable to find the System.Console.WriteLine() method. Disabling debug injection. To enable debug injection, ensure the path to mscorlib is passed to '{0}' with the -L option.</value>
<comments>{0} - name of the command line tool
The following terms should not be translated: System.Console.WriteLine(), mscorlib, -L.</comments>
```